### PR TITLE
merchants/tools index: remove localmonero

### PR DESCRIPTION
**Pull request by plowsof**

### CODE REVIEW

**Comment by plowsof** [comment]
https://moneroblocks.info/ redirects to https://localmonero.co/blocks/ will update this PR to remove that from tools also

**Comment by luigi1111** [comment]
Is the block explorer being wound down too? Sad to lose that piece of history (moneroblocks old school lore).

**Comment by plowsof** [comment]
@luigi1111 im not sure how long this has been happening but https://moneroblocks.info/ redirecting to localmonero shows me that the service has already wound down OR... it happened only yesterday and theyre doing some maintenance, i will try to confirm

**Comment by plowsof** [comment]
https://libera.monerologs.net/monero-community/20240721#c400230

Alex confirms the block explorer will continue and provided an onion URL also.

Removing moneroblocks because its a duplicate and leaving localmonero?

**Review by nahuhh — APPROVED** [review]

File: _i18n/en.yml [hunk 1]
```diff
@@ -1021,7 +1020,6 @@ tools:
 
   network: Network
   monerohash-nodes: Monero node map
-  localmonero-blocks: Explorer and Statistics
```
Comment (HardenedSteel) [inline _i18n/en.yml:1024]: if the blockchain explorer will continue this line should stay.

Comment (plowsof) [reply _i18n/en.yml:1024]: it's a block explorer, this PR moves it under block explorers. a mirror of the "moneroblocks" that was listed under block explorers. it is now redundant text

**Review by HardenedSteel — APPROVED** [review]

**Pull request merged**